samples/tests/playwright.config.ts

```diff
@@ -17,6 +17,7 @@ const config: PlaywrightTestConfig = {
       use: {
         browserName: 'chromium',
         permissions: ['notifications', 'camera', 'microphone'],
+        viewport: { width: 900, height: 900 },
```

Comment:
Add a comment above saying why this is needed, e.g.: "Large enough to contain all the UI elements added in ComponentExamples sample".